packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -115,6 +115,8 @@ export const LivenessCameraModule = (
 
   const isFaceMovementChallenge =
     useLivenessSelector(selectChallengeType) === FACE_MOVEMENT_CHALLENGE.type;
+  const isMobileFaceMovementAndLightChallenge =
```

Comment:
Can we flip this logic to be more readable, something like 
`const allowSelectableDevices = !isMobile || (isMobile && isFaceMovementChallenge)`
then add a comment why we only support desktop or mobile and facemovement?